src/style-spec/reference/v8.json

```diff
@@ -188,8 +188,8 @@
       "sdk-support": {
         "basic functionality": {
           "js": "Not yet available",
-          "android": "Not yet available",
-          "ios": "Not yet available",
+          "android": "9.3.0",
+          "ios": "5.10.0",
           "macos": "Not yet available"
```

Comment:
You can omit platforms from the json where the property isn't available, and the style spec docs page will automatically populate with "Not yet supported." So delete `"js:"` and `"macos:"` from this commit

See example at https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/#paint-fill-extrusion-fill-extrusion-vertical-gradient and https://github.com/mapbox/mapbox-gl-js/blob/4b8dd318a80d73d63633129d8345fbc8f99fc783/src/style-spec/reference/v8.json#L4170-L4180